parent/pom.xml

```diff
@@ -493,6 +493,20 @@
             <version>${version.hibernate.entitymanager}</version>
          </dependency>
          <dependency>
+            <groupId>org.jboss.shrinkwrap.resolver</groupId>
+            <artifactId>shrinkwrap-resolver-bom</artifactId>
+            <version>${version.shrinkwrapResolver}</version>
+            <type>pom</type>
+            <scope>import</scope>
+         </dependency>
+         <!-- This is a work around for https://issues.jboss.org/browse/SHRINKRES-238 -->
+         <!-- After fixing the issue above, the dependency below can be safely removed -->
```

Comment:
The problem is that we import `wildfly-core-parent` that has a dependency to `maven-aether-provider`. Here's the relevant part of the output of executing `mvn -X dependency:tree`:

```
org.apache.maven:maven-aether-provider:jar:3.2.1:test (version managed from 3.2.5 by org.wildfly.core:wildfly-core-parent:2.2.0.Final)
```

So Maven figures out that `maven-aether-provider` provided by WF Core is [closer](https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html#Transitive_Dependencies) than the one provided by `shrinkwrap-resolver-bom`. The only way to override this (at least to my knowledge) is to add a specific version to `dependencyManagement`. 

Sucks.. I know..